force-app/main/default/pages/GE_TokenizeCard.page

```diff
@@ -208,15 +208,17 @@
         }
 
         function buildLightningOrigin() {
-            const hostname = window.location.hostname;
-            const arr = hostname.split(".");
-            let domain = arr[0].replace('--npsp', '');
-            domain = domain.replace('--c', '');
             const siteBaseURL = '{!$Site.BaseUrl}';
+            let orgUrl = '{!OrgUrl}';
+            const sforceBaseUrl = '{!SFBaseUrl}';
+
+            if(sforceBaseUrl.includes("vf.force.com")){
```

Comment:
I'm not super confident that this logic will consistently get us what we want but it allows for dynamic URL creation

Reply:
I think we should write a comment clearly stating why we went this route here.

Reply:
That is a very good point, I can do that!